packages/docs-reanimated/docs/layout-animations/layout-transitions.mdx

```diff
@@ -4,195 +4,328 @@ sidebar_position: 3
-  controls="controls"
-  muted="muted"
-  height="600"
+- `.duration(duration: number)` sets length of the animation (in milliseconds). Defaults to '300'.
+- `.delay(duration: number)` is the delay before the animation starts (in milliseconds). Defaults to `0`.
+- `.easingX(easing: EasingFunction)` defines easing animation for x-position.
+- `.easingY(easing: EasingFunction)` defines easing animation for y-position.
+- `.easingWidth(easing: EasingFunction)` defines easing animation for width.
+- `.easingHeight(easing: EasingFunction)` defines easing animation for height.
+- `.reduceMotion(reduceMotion: ReduceMotion)` determines how the animation responds to the device's reduced motion [accessibility](/docs/guides/accessibility) setting.
+- `.withCallback(callback: (finished: boolean) => void)` is the callback that will fire after the animation ends. Sets `finished` to `true` when animation ends without interruptions, and `false` otherwise.
+
+### Example
+
+<ExampleVideo
+  sources={{
+    android:
+      '/react-native-reanimated/recordings/layout_transitions/android_curved_transition.mov',
+    ios: '/react-native-reanimated/recordings/layout_transitions/ios_curved_transition.mov',
+  }}
 />
 
-### Curved Transition
+## Entry/Exit Transition
 
-Curved transition, enables to animate each position and dimension with different easing which makes components animation curved.
+Entry/Exit transition lets you choose layout animations for entering and exiting. It combines both of them into a one and applies to the layout. Moreover, instead of picking currently available animations, you are welcome to create your own one.
 
-#### Modifiers
+### Reference
 
-- `duration` (in ms) default: 300
-- `delay` (in ms) default: 0
-- `withCallback` callback that will fire after the exit animation ends
-- `randomDelay` randomize delay of the animation between 0 and provided delay ( or 1000ms if delay not provided )
-- `easingX` provides Easing for x-position ( default: `Easing.in(Easing.ease)`)
-- `easingY` provides Easing for y-position ( default: `Easing.out(Easing.ease)`)
-- `easingWidth` provides Easing for width ( default: `Easing.in(Easing.exp)`)
-- `easingHeight` provides Easing for height ( default: `Easing.out(Easing.exp)`)
-- `reduceMotion` determines how the animation responds to the device's reduced motion accessibility setting.
+```js
+import { EntryExitTransition } from 'react-native-reanimated';
 
-#### Example
+function App() {
+  return <Animated.View layout={EntryExitTransition} />;
+}
+```
 
-<video
-  src="https://user-images.githubusercontent.com/48885911/134479266-5c7342f5-7453-4389-a6c5-5070e3673822.mov"
-  controls="controls"
-  muted="muted"
-  height="600"
-/>
+### Modifiers
 
-### Entry/Exit Transition
+```javascript
+EntryExitTransition.duration(1000)
+  .delay(500)
+  .reduceMotion(ReduceMotion.Never)
+  .withCallback((finished) => {
+    console.log(`finished without interruptions: ${finished}`);
+  });
+```
```

Comment:
I'd suggest adding some .exiting and .entering modifiers here so that users can see how it should be done